threatstream.json

```
@@ -8251,17 +8251,10 @@
             "read_only": false,
             "parameters": {
                 "observable_type": {
-                    "description": "Type of observable to import",
+                    "description": "Type of observable to import, default types: [ip, domain, url, email, hash]. Custom types can be provided if they are available on ThreatStream instance",
```

Not sure if current description is clear enough. Maybe we can improve it a bit:
```suggestion
                    "description": "Type of observable to import, types supported by default: [ip, domain, url, email, hash]. Custom types can also be provided if they are available on ThreatStream instance",
```